data-reconciliation-app/README.md

```diff
@@ -26,19 +33,62 @@ To get started run `cd data-reconciliation-app && make test` to run the applicat
 │   └── adrs            All Architecture decision records (ADR)
 │
 ├── governance
-│   └── constitution    CCF network constitution files
-│   └── nodes           CCF network nodes configs
-│   └── scripts         All governance scripts
-│   └── vote            Contains proposal voting acceptance and rejection logic
+│    └── constitution    CCF network constitution files
+│    └── nodes           CCF network nodes configs
+│    └── scripts         All governance scripts
+│    └── vote            Contains proposal voting acceptance and rejection logic
 │
 └── src                 Application source code
+│    └── endpoints      Application endpoints
+│    └── models         Domain models
+│    └── repositories   Data repositories
+│    └── services       Domain services
+│    └── utils          utility classes
 │
 └── test
-│    └── e2e-test       Application end to end tests
-│    └── unit-test      Application unit tests
+     └── e2e-test       Application end to end tests
+     └── unit-test      Application unit tests
+
+```
+
+## Getting Started
+
+To get started and run the application locally, we will start with setting up the environment.
+
+```bash
+# setup the environment
+git clone https://github.com/microsoft/ccf-app-samples # Clone the samples repository
+code .                                                 # open samples repository in Visual studio code
+
+# In the VScode terminal window
+cd data-reconciliation-app                             # Navigate to reconciliation sample folder
+make build                                             # Build and create the application deployment bundle
 ```
 
-### Running Locally
+Now the environment is ready, and there are several scenarios that could be executed at this stage.
+
+- **Run the application's [e2e-tests](./test/test.sh) in a sandbox environment in the interactive mode**
+  - `make demo`
+
+- **Run the application's [e2e-tests](./test/test.sh) in a sandbox environment**
+  - `make test`
+
+- **Run the application's [e2e-tests](./test/test.sh) on a Docker container running a virtual environment (sandbox)**
```

Comment:
```suggestion
- **Run the application's [e2e-tests](./test/test.sh) on a Docker Container running a virtual (simulated) environment**
```

If I'm not mistaken docker containers' flow have no relationship with sandbox

Reply:
Should we document `test-docker-enclave`? 

if so, my suggestion:

**Run the application's [e2e-tests](./test/test.sh) on a Docker Container running in a enclave environment**
- Requires a CCF-compatible hardware.
- `make test-docker-virtual`